src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.file.FileReader;
+import core.basesyntax.file.FileReaderImpl;
+import core.basesyntax.file.FileWriter;
+import core.basesyntax.file.FileWriterImpl;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.OperationStrategy;
+import core.basesyntax.operation.OperationStrategyImpl;
+import core.basesyntax.operation.handler.BalanceOperation;
+import core.basesyntax.operation.handler.PurchaseOperation;
+import core.basesyntax.operation.handler.ReturnOperation;
+import core.basesyntax.operation.handler.SupplyOperation;
+import core.basesyntax.reportgenerator.ReportGenerator;
+import core.basesyntax.reportgenerator.ReportGeneratorImpl;
+import core.basesyntax.shopservice.ShopService;
+import core.basesyntax.shopservice.ShopServiceImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String FILE_TO_READ = "src/main/java/resources/reportToRead.csv";
+    private static final String FILE_TO_WRITE = "src/main/java/resources/finalReport.csv";
```

Comment:
```suggestion
    private static final String FILE_TO_READ = "src/main/resources/reportToRead.csv";
    private static final String FILE_TO_WRITE = "src/main/resources/finalReport.csv";
```
here is a valid resources path